code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm

```diff
@@ -34,124 +31,257 @@
+				carbon.apply_armoured_damage(get_xeno_damage_slash(carbon, base_damage_aoe + damage_scale_aoe * predalienbehavior.kills), ARMOR_MELEE, BRUTE, "chest", 20)
+			playsound(owner, 'sound/voice/predalien_death.ogg', 75, 0, status = 0)
+		REMOVE_TRAIT(xeno, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Eviscerate"))
+		xeno.anchored = FALSE
+		apply_cooldown()
+		return ..()
 
-	if(!check_plasma_owner())
+	//single target checks
+	if (xeno.can_not_harm(target))
+		to_chat(xeno, SPAN_XENOWARNING("We must target a hostile!"))
 		return
 
-	if(!do_after(xeno, activation_delay, INTERRUPT_ALL, BUSY_ICON_GENERIC))
-		to_chat(xeno, "Keep still whilst trying to smash into the ground")
-
-		var/real_cooldown = xeno_cooldown
-
-		xeno_cooldown = 3 SECONDS
-		apply_cooldown()
-		xeno_cooldown = real_cooldown
+	if (get_dist_sqrd(target, xeno) > 2)
+		to_chat(xeno, SPAN_XENOWARNING("[target] is too far away!"))
 		return
 
-	if(!check_and_use_plasma_owner())
+	var/mob/living/carbon/carbon = target
+
+	if (carbon.stat == DEAD)
+		to_chat(xeno, SPAN_XENOWARNING("[carbon] is dead, why would we want to touch them?"))
 		return
+	if(targetting == SINGLETARGETGUT) // single target
+		ADD_TRAIT(carbon, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Devastate"))
+		apply_cooldown()
 
-	playsound(xeno.loc, pick(smash_sounds), 50, 0, status = 0)
-	xeno.visible_message(SPAN_XENOHIGHDANGER("[xeno] smashes into the ground!"))
+		ADD_TRAIT(xeno, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Devastate"))
+		xeno.anchored = TRUE
 
-	xeno.create_stomp()
+		if (do_after(xeno, activation_delay, INTERRUPT_ALL | BEHAVIOR_IMMOBILE, BUSY_ICON_HOSTILE))
+			xeno.visible_message(SPAN_XENOHIGHDANGER("[xeno] rips open the guts of [carbon]!"), SPAN_XENOHIGHDANGER("We rip open the guts of [carbon]!"))
+			carbon.spawn_gibs()
+			playsound(get_turf(carbon), 'sound/effects/gibbed.ogg', 50, 1)
+			carbon.apply_effect(get_xeno_stun_duration(carbon, 0.5), WEAKEN)
+			carbon.apply_armoured_damage(get_xeno_damage_slash(carbon, base_damage + damage_scale * predalienbehavior.kills), ARMOR_MELEE, BRUTE, "chest", 20)
 
-	for(var/mob/living/carbon/carbon in oview(round(behavior.kills * 0.5 + 2), xeno))
-		if(!xeno.can_not_harm(carbon) && carbon.stat != DEAD)
-			ADD_TRAIT(carbon, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Smash"))
+			xeno.animation_attack_on(carbon)
+			xeno.spin_circle()
+			xeno.flick_attack_overlay(carbon, "tail")
 
-			if (ishuman(carbon))
-				var/mob/living/carbon/human/human = carbon
-				human.update_xeno_hostile_hud()
+		playsound(owner, 'sound/voice/predalien_growl.ogg', 50, 0, status = 0)
 
-			addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(unroot_human), carbon, TRAIT_SOURCE_ABILITY("Smash")), get_xeno_stun_duration(carbon, freeze_duration))
+		REMOVE_TRAIT(xeno, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Devastate"))
+		xeno.anchored = FALSE
+		unroot_human(carbon, TRAIT_SOURCE_ABILITY("Devastate"))
 
+		xeno.visible_message(SPAN_XENODANGER("[xeno] rapidly slices into [carbon]!"))
```

Comment:
Can include the self message here e.g. `SPAN_XENODANGER("We rapidly slice into [carbon]!"`